layer/deploy_layer.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/sh
 VERSION=$1
```

Comment:
it would be better if we could get version by parsing `setup.py` with grep as did in for Java by parsing `pom.xml`